exoseq.nf

```diff
@@ -47,15 +59,15 @@ Other options:
 --project                      Uppnex project to user for SLURM executor
 
 For more detailed information regarding the paramaters and usage refer to package
-documentation at https://github.com/SciLifeLab/NGI-EXOseq
+documentation at https:// github.com/SciLifeLab/NGI-ExoSeq
 """
 
-//Variables and defaults
+// Variables and defaults
 params.help = false
 params.reads = false
 params.genome = false
 params.clusterOptions = false
-params.project = 'ngi2016003'
+params.project = false
 params.outdir = './results'
```

Comment:
Just for understanding, you mean _'it **should** be defined'_ or _'it is **better** to be defined'_. 

My understanding is the value of same parameter is weighted as `param_in_command_line > param_in_config_file > param_in_main_script`. So since here 'outdir' is mentioned in `nextflow.config` I dont think what we mention in `exoseq.nf` is gonna matter.